Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,278 @@
+            self.legendItems = legendItems.sorted { $0.name < $1.name }
+        }
+        
+        // MARK: Network Association Graphics
+        
+        /// Creates graphics for the associations and content in the
+        /// container element.
+        /// - Parameter feature: The container element's feature.
+        func handleIdentifiedFeature(_ feature: ArcGISFeature) async throws {
+            let containerElement = network.makeElement(arcGISFeature: feature)!
+            let contentElements = try await makeContentElements(containerElement: containerElement)
+            let contentGraphics = try await makeGraphicsForContentElements(contentElements, within: containerElement)
+            graphicsOverlay.addGraphics(contentGraphics)
+            
+            if contentGraphics.count == 1 {
+                await setStatusMessage("This feature contains no associations.")
+            } else {
+                await setStatusMessage("Contained associations are shown.")
+            }
+            
+            if let extent = graphicsOverlay.extent {
+                let associationsGraphics = try await makeGraphicsForAssociations(within: extent)
+                let boundingBoxGraphic = Graphic(geometry: GeometryEngine.buffer(around: extent, distance: 0.05)!, symbol: boundingBoxSymbol)
+                graphicsOverlay.addGraphics(associationsGraphics)
+                graphicsOverlay.addGraphic(boundingBoxGraphic)
+            }
+        }
+        
+        /// Creates content elements within the container element.
+        /// - Parameter containerElement: The container element.
+        /// - Returns: An array of utility elements in the container.
+        private func makeContentElements(containerElement: UtilityElement) async throws -> [UtilityElement] {
+            // Gets the containment associations from the element to display its content.
+            let containmentAssociations = try await network.associations(for: containerElement, ofKind: .containment)
+            // Determines the relationship of each element and add it to the content elements.
+            let contentElements: [UtilityElement] = containmentAssociations.map { association in
+                if association.fromElement.objectID == containerElement.objectID {
+                    return association.toElement
+                } else {
+                    return association.fromElement
+                }
+            }
+            return contentElements
+        }
+        
+        /// Creates the graphics for the utility elements within a container.
+        /// - Parameters:
+        ///   - contentElements: The elements within the container.
+        ///   - containerElement: The container element.
+        /// - Returns: An array of graphics.
+        private func makeGraphicsForContentElements(_ contentElements: [UtilityElement], within containerElement: UtilityElement)  async throws -> [Graphic] {
```

Comment:
Since this is a private API, it is fine that it doesn't follow the guideline that factory methods should not form a phrase starting with the base name. But it should follow the universal guideline that when the first argument forms part of a prepositional phrase, it should have an argument label. So a better name would be `makeGraphics(for:within:)`.